docs/CHANGELOG.rst

```diff
@@ -4,6 +4,29 @@ Changelog
 
 .. current developments
 
+v1.3.1
+====================
+
```

Comment:
```suggestion
Bugfix release - Improved error handeling and code cleanup.

We are also dropping offical support for MacOSX-x86_64.
Anyy platform-specific bugs will be addressed when possible, but as a low priority.
```

Reply:
(I updated your comment to fix typos because I wanted to used "Commit suggestion")